h2o-core/src/main/java/water/H2O.java

```diff
@@ -521,6 +527,14 @@ else if (s.matches("login_conf")) {
         i = s.incrementAndCheck(i, args);
         ARGS.login_conf = args[i];
       }
+      else if (s.matches("login_conf")) {
```

Comment:
The conf file for ldap etc. services. It's been there for a long time, seems I duplicated that else if by accident during one of the merges. Removed.